openslides/utils/cache.py

```diff
@@ -0,0 +1,199 @@
+from django.core.cache import caches, cache
+from collections import defaultdict
+from channels import Group
+from channels.sessions import session_for_reply_channel
+
+
+class WebsocketUserCache:
+    """
+    Caches the reply channel names of all open websocket connections to the
```

Comment:
`Caches the reply channel names of all open websocket connections. The id of the user that that opened the connection is used as reference.`